virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -10,344 +10,282 @@ ms.service: windows-containers
 
+### To use DHCP for IP assignment on a virtual container host enable MACAddressSpoofing 
+If the container host is virtualized, and you wish to use DHCP for IP assignment, you must enable MACAddressSpoofing on the virtual machine's network adapter. Otherwise, the Hyper-V host will block network traffic from the containers in the VM with multiple MAC addresses. You can enable MACAddressSpoofing with this PowerShell command:
 ```none
-C:\> docker network create -d transparent -o com.docker.network.windowsshim.networkname=MyTransparentNetwork MyTransparentNetwork
+PS C:\> Get-VMNetworkAdapter -VMName ContainerHostVM | Set-VMNetworkAdapter -MacAddressSpoofing On
 ```
+### Creating multiple transparent networks on a single container host
+If you wish to create more than one transparent network you must specify to which (virtual) network adapter the external Hyper-V Virtual Switch should bind. To specify the interface for a network, use the following syntax:
 
-#### Example: Default HNS Naming Behavior
+--- @Jason: Is this syntax right?
+```
+# General syntax:
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.interface=<INTERFACE NAME> <NETWORK NAME> 
 
-To put the behavior of this naming option into context, the screen capture below demonstrates how a network is named by the HNS service when this naming option is *not* used. In this example, the name of the network, "MyTransparentNetwork" is visible to Docker, as shown via the `docker network ls` command. The name of the network is not visible, however, to the HNS service, as shown via the `Get-ContainerNetwork` Windows PowerShell command; instead, a large alphanumeric name for the network has been automatically generated by HNS.
+# Example:
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.interface="Ethernet 2" myTransparent2
+```
 
-><figure>
-  <img src="media/SpecifyName_Capture.PNG">
-  <figcaption>Example: The name of a network is <i>not</i> specified to the HNS service. </figcaption>
-</figure>
+### Remember to specify *--subnet* and *--gateway* when using static IP assignment
+When using static IP assignment, you must first ensure that the *--subnet* and *--gateway* parameters are specified when the network is created. The subnet and gateway IP address should be the same as the network settings for the container host - i.e. the physical network.
 
-#### Example: Specifying the Name of a Network to the HNS Service
 
-When the `-o com.docker.network.windowsshim.networkname=<network name>` *is* used, on the other hand, the HNS service uses the specified name instead of a generated name. This behavior is demonstrated in the screen capture below.
+### DHCP IP assignment not supported with L2Bridge networks
+--- @Jason: This first says only static is supported then only says dynamic is supported. What's the take-home I need to try to capture on this one?
```

Comment:
Dynamic (DHCP) is not supported with l2bridge mode since there is no unique identifier (i.e. MAC address) sent on the physical network. All container endpoints will appear they are using the same MAC address. Therefore, in l2bridge mode (without any sort of external cloud plugin - e.g. used by Azure) users must specify a static IP address.